DEVELOPMENT.md

```diff
@@ -0,0 +1,64 @@
+
+## One Time Setup
+
+Install goose for db migrations:
+
+```
+$ go install github.com/pressly/goose/v3/cmd/goose@latest
+```
+
+## Create and Populate PostgreSQL Database
+
+Launch postgresql: 
+
+```bash
+podman run --name sippy-postgres -e POSTGRES_PASSWORD=password -p 5432:5432 -d quay.io/enterprisedb/postgresql
+```
+
+TODO: podman run postgres as a regular user may require disabling selinux on Fedora 35. Needs investigation.
```

Comment:
Do you know what the selinux denials were? I don't seem to have that problem:

```
[stbenjam@stbenjam sippy]$ getenforce
Enforcing
[stbenjam@stbenjam sippy]$ cat /etc/redhat-release 
Fedora release 35 (Thirty Five)
```

Reply:
do you run podman as root? I've been doing it as a regular user.

May 17 08:32:07 berlinetta audit[22918]: AVC avc:  denied  { read } for  pid=22918 comm="docker-entrypoi" path="/usr/lib64/librt-2.28.so" dev="dm-0" ino=3829261 scontext=system_u:system_r:container_t:s0:c7,c741 tcontext=unconfined_u:object_r:user_home_t:s0 tclass=file permissive=0
May 17 08:32:07 berlinetta sippy-kube-postgres[22915]: /usr/bin/coreutils: error while loading shared libraries: librt.so.1: cannot change memory protections

Reply:
Yes, always as a regular user.

Reply:
Maybe try https://github.com/containers/podman/issues/3234#issuecomment-1043188614 ?